sklearn/model_selection/_validation.py

```diff
@@ -129,6 +128,7 @@ def cross_val_score(estimator, X, y=None, groups=None, scoring=None, cv=None,
     X, y, groups = indexable(X, y, groups)
 
     cv = check_cv(cv, y, classifier=is_classifier(estimator))
+    cv_iter = cv.split(X, y, groups)
```

Comment:
Just as a sign of best practice denoting that we enlist the indices and store them as soon as we check the `cv` param...